tools/myst/enc/gcov.c

```diff
@@ -0,0 +1,517 @@
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <limits.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+
+#include <myst/eraise.h>
+
+#include "myst_t.h"
+
+#ifdef MYST_ENABLE_GCOV
+
+#define MYST_GCOV_STDERR ((FILE*)0x67001b41aafb4224)
```

Comment:
Should we remove this and include <myst/gcov.h> to get the definition of ``MYST_GCOV_STDERR``

Reply:
I know is was my fault. :)

Reply:
Added this